runtime/lua/vim/_defaults.lua

```diff
@@ -964,6 +964,14 @@ do
       end
     end,
   })
+
+  vim.api.nvim_create_autocmd('TextYankPost', {
+    group = vim.api.nvim_create_augroup('nvim.yank', {}),
+    desc = 'Highlight yanked text',
+    callback = function()
+      vim.hl.on_yank()
```

Comment:
is 150 ms really enough? I have been using 300 ms

Reply:
That's the default (which I've been using, no surprise) -- we can of course change that if we think 300ms is better. (I'm fine with either, but I don't think we should enable it with non-default settings.)

Reply:
let's try changing the default to 300ms

Reply:
Chiming in here. 300 feels really slow to me... I have been using 100, but am more than happy with the default just staying 150